docs/01_nodeos/06_logging/20_third_party_logging/index.md

```diff
@@ -0,0 +1,15 @@
+---
+content_title: Third-Party Logging And Tracing Integration
+link_text: Third-Party Logging And Tracing Integration
```

Comment:
This left nav menu item renders too long (might wrap to next line). Consider:

link_text: Third-Party Logging And Tracing